src/main/java/ca/ciccc/silverBullet/gameBoard/GridBoard.java

```diff
@@ -252,5 +252,14 @@ public boolean tryShoot (Player playerShooting){
     public void removePlayer(Player playerToRemove){
         GameScene.instance.getChildren().remove(playerToRemove.getPlayerNode());
     }
+
+    public boolean areAllFull(){
+        for (Player p : players){
+            if(!p.isActionsFull()){
+                return false;
+            }
+        }
+        return true;
+    }
```

Comment:
Easy way to do this
```
return this.players.stream().allMatch(Player:: isActionsFull)
```

Reply:
Cool. Changed as well.

Reply:
4b1540ace1d5dc853e226e6ad0d35fdbc42b65ac

Reply:
00e25cde836265381f43360a33c9c9eb5646139d